frontend/src/component/Activity/Preview.styl

```diff
@@ -0,0 +1,61 @@
+@import "~tracim_frontend_lib/src/css/Variable.styl"
+
+previewBorderColor = mediumGrey
+previewHeight = 300px
+linkColor = #007bff // This color comes from bootstrap
+
+.activityFeed__preview
+  box-sizing content-box
+  position relative
+  overflow hidden
+  border 1px solid previewBorderColor
+  background-color white
+  max-height previewHeight
+
+  &__overflow
+    background-image:
+      linear-gradient(to top, rgba(0, 0, 0, 0.125), white),
+      radial-gradient(ellipse at bottom, rgba(0, 0, 0, 0.25), rgba(0, 0, 0, 0))
+
+    background-size 100% 25%
+    background-repeat no-repeat
+    background-position bottom
+
+  & > a
+    display flex
+    min-height 100%
+    color inherit
+    overflow hidden
+
+    & > *
+      display flex
+      flex 1
+      align-items center
+      justify-content center
+      flex-direction column
+      overflow hidden
+
+  &__html
+    text-align justify
+
+    a-in-preview
+      color linkColor
+
+  &__unavailable
+    display:block
+    text-align:center
+
+    & > a
+      display:block
+
+    .fa
+      font-size previewIconFontSize
+
+    .fa, span
+      display inline-block
+      padding 0.5ex
```

Comment:
Normally we use px

Reply:
Having something relative to the font seems more appropriate here, no?

Reply:
it would but it require to have an architecture based on that approach which we don't.
Using it only in a few places will result in some unexpected behavior

Reply:
I changed this to padding: 10px locally